source/Octopus.Tentacle/Services/Capabilities/CapabilitiesService.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections.Generic;
+using Octopus.Tentacle.Contracts;
+using Octopus.Tentacle.Contracts.Capabilities;
+
+namespace Octopus.Tentacle.Services.Capabilities
+{
+    [Service]
+    public class CapabilitiesService : ICapabilitiesService
+    {
+        public CapabilitiesResponse GetCapabilities()
+        {
+            return new CapabilitiesResponse(new List<string>());
```

Comment:
Should we have a test that test the happy path e.g. call the service and it returns the empty list. This will then return some items when they get added later on?

Reply:
Added